sympy/functions/special/gamma_functions.py

```diff
@@ -96,7 +96,7 @@ class gamma(Function):
     loggamma: Log Gamma function.
     digamma: Digamma function.
     trigamma: Trigamma function.
-    beta: Euler Beta function.
+    sympy.functions.special.beta_functions.beta: Euler Beta function.
```

Comment:
`MusculotendonBase` has an attribute called `beta`. For some reason Sphinx was getting confused about whether to link to the `beta` function from the `functions.special` module or the `beta` attribute on `MusculotendonBase` in the `physics.biomechanics` module. The solution was to provide the full paths here so that Sphinx knew which of the two to use.

What's not clear to me is why Sphinx was getting confused between a class and an attribute on a class with the same name. I'd have thought that it shouldn't be considering attributes in this case, but it was.

Reply:
Ok. Let's merge this!